modules/ROOT/pages/configure-multiple-credential-providers.adoc

```diff
@@ -0,0 +1,116 @@
+| Business Group Administrator | Be notified about any possible interruption to API consumers when I am about to change any IdP configuration.  | Access Management | None
+| Master organization Administrator | Change the default IdP across all environments and across all business groups. | Access Management | Resetting the default IDP updates all environments across all business groups to a selected IDP, without affecting existing client applications. This way, when a new IdP is created for the organization, all environments are updated with just one change.
+| API Manager Administrator | Review which client provider an application belongs to on the client administrator page to be able to differentiate between different applications with the same name. | Access Management | None
+| API Owner | Configure my API in API Manager to automatically map to a default IdP configured by the business group administrator. I can also select a different IdP, if available | API Manager | None
+| API Owner | When promoting or importing an API between environments, specify an IdP from the target environment. | API Manager | If there is just one (default) IdP specified for the target environment, it is automatically assigned to the API on promotion. 
+| Application User | Request access to an API based on the API instance that exists in a specific environment. | Exchange | When you select the API for which you want to request access, you must first choose the API instance to which you are requesting access. The API instance is filtered based on the environment.
+|===
+
+=== Configure Multiple Client Providers
+
+To configure multiple external client providers:
+
+* Step 1: From *Access Management*, first enable xref:access-management::managing-api-clients.adoc[client management].
++
+As an Anypoint Administrator, you must first enable clients applications to be authenticated using OAuth for the client providers that you want to implement in your organization.
+* Step 2: From *Access Management*, xref:access-management::environments.adoc#add-an-idp-to-an-existing-environment[assign client providers] to environments and business groups.
++
+As a Business Administrator, you must then configure multiple client providers and assign them to specific business organizations and environments.
+* Step 3: From *API Manager*, xref::create-instance-task.adoc[create or modify] APIs using the new IdPs.
++
+As an API Owner, you can create an API, import an API from Exchange or modify an API in API Manager. Optionally, you can promote an API from one environment to another, and import or export an API, as described in the user flow diagram in the previous section. 
```

Comment:
```suggestion
As an API Owner, you can create an API, import an API from Anypoint Exchange, or modify an API in API Manager. Optionally, an owner can promote an API from one environment to another, and import or export an API, as described in the user flow diagram in the previous section. 
```
Replace reference to "the user flow diagram in the previous section" with the section title.

Reply:
Instead of owner I prefer to leave the `you`

Reply:
Made changes to match the 3rd person reference.